LayoutTests/imported/w3c/web-platform-tests/css/css-cascade/all-prop-initial-xml-expected.txt

```diff
@@ -288,6 +288,8 @@ PASS scroll-padding-top
 PASS scroll-snap-align
 PASS scroll-snap-stop
 PASS scroll-snap-type
+FAIL scroll-timeline-axis assert_equals: expected "none" but got "block"
```

Comment:
This suggests a problem with `scroll-timeline-axis`, `none` is not a valid value.

Other test regressions are similar.